### PR TITLE
Fix build C libraries inside kphp

### PR DESCRIPTION
**Pull request by andreylzmw**
fix issue https://github.com/VKCOM/kphp/issues/804

### CODE REVIEW

File: builtin-functions/_functions.txt [hunk 1]
```diff
@@ -1622,3 +1622,6 @@ class DateTimeImmutable implements DateTimeInterface {
 }
 
 function getenv(string $varname = '', bool $local_only = false): mixed;
+
+/* mbstring */
+function mb_convert_encoding(string $str, string $to, string $from): string;
```
Comment (Danil42Russia) [inline builtin-functions/_functions.txt:1627]: Is this really relevant to this PR?

Comment (andreylzmw) [reply builtin-functions/_functions.txt]: Sorry, no, fixed now.

File: cmake/external-libraries.cmake [hunk 2]
```diff
@@ -54,6 +54,8 @@ else()
     add_link_options(-L${kphp-timelib_SOURCE_DIR}/objs)
 endif()
 
+add_compile_options(-Wno-redundant-move)
```
Comment (Ramzeeset) [inline cmake/external-libraries.cmake:57]: Let's move the `kphp-timelib` installation to the top, to use the flag to build other libraries. Also let's add an explanatory comment why this option is added here

Comment (andreylzmw) [reply cmake/external-libraries.cmake]: Ok, done

**Comment by troy4eg** [comment]
resolved in https://github.com/VKCOM/kphp/pull/808
